torch/_dynamo/trace_rules.py

```diff
@@ -244,6 +244,8 @@
     "torch._C.set_autocast_xla_dtype": SkipFunctionVariable,
     "torch._C.set_autocast_xla_enabled": SkipFunctionVariable,
     "torch.resize_as_": SkipFunctionVariable,
+    "torch._functorch.vmap._add_batch_dim_in_graph": TorchInGraphFunctionVariable,
+    "torch._functorch.vmap._remove_batch_dim_in_graph": TorchInGraphFunctionVariable,
```

Comment:
On second thought, I really don't like the special names. Especially if these will be BC-breaking to change.

Can we name these something sane?
- torch._functorch.vmap._add_batch_dim
- torch._functorch.vmap._remove_batch_dim
- torch._functorch.vmap._vmap_increment_nesting
- torch._functorch.vmap._vmap_decrement_nesting